Jenkinsfile

```diff
@@ -0,0 +1,18 @@
+node {
+  stage('SCM CheckOut') {
+git 'https://github.com/UganderD/TestJavaApp.git'
+    
+
+}
+stage('Compile-Package'){
+//Get maven home path
+def mvnHOME = tool name: 'M2_HOME', type: 'maven'
+
+sh "${mvnHOME}/bin/mvn package"
+}
```

Comment:
Also, try to add more CI steps like:-
- Code Quality
- Code Coverage
- Unit Testing